### DISCUSSION TITLE
Github Ubuntu instance no libomp please install

### DISCUSSION THREAD

**Discussion by mu578** [community/community#57274] [Actions] [answered]
### Select Topic Area

Bug

### Body

usr/bin/ld: cannot find -lomp: No such file or directory please apt-get libomp-dev or use `llvm.sh` to install current packages ; then allow symlinks to clang/clang++/omp in usr paths to right install. Thank you.

**Comment by backtoaldo** [comment]
Try this:

Update your Server:
`sudo apt update && sudo apt upgrade`

Install libomp-dev:
`sudo apt install libomp-dev`

Verify Installation:
`dpkg -L libomp-dev`

If the simple solution doesn't work, could you let me know which version of Ubuntu you currently use?

**Comment by mu578** [comment, marked as answer]
Not my server it is github ci instances.

<img width="1874" alt="Screenshot 2023-06-05 at 6 12 49 PM" src="https://github.com/community/community/assets/19998532/e82fd2a4-53c5-4af9-baf7-7802d8b52b59">

**Reply by mucmec-fohfex-jiwKi0** [reply]
I am not sure how to answer this question without asking the question myself

**Comment by mucmec-fohfex-jiwKi0** [comment]
#F can CB BBC I B C B B C vomiting and I have a headache from the last day I was at a doctors appointment

**Comment by mu578** [comment]
closing to avoid random junkies. BTW, @backtoaldo https://github.com/mu578/mu0/blob/master/mk/inc/mu0_toolchain.mk#L279 still ugly to run ci proper.
